app/views/layouts/application.html.erb

```diff
@@ -5,11 +5,54 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
-    <%= yield %>
+    <header>
+      <nav class="navbar navbar-expand-md navbar-light bg-light">
+        <div class="container-lg">
+          <%= link_to 'Private Events', root_path, class: 'navbar-brand fw-bold rounded-pill border border-3 px-3' %>
+          <button class="navbar-toggler border-0" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavLinks" aria-controls="navbarNavLinks" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNavLinks">
+            <div class="navbar-nav ml-auto">
+              <%= link_to 'New Event', new_event_path, class: 'nav-link border-right fw-bold' %>
+              <% if user_signed_in? %>
+                <%= link_to user_path(current_user), class: 'nav-link fw-bold' do %>
+                  Hello, <%= current_user.name %>
+                <% end %>
+                <%= link_to 'Logout', session_path(current_user.id), method: :delete, class: 'nav-link text-danger' %>
+              <% else %>
+                <%= link_to 'Login', new_session_path, class: 'nav-link' %>
+                <%= link_to 'Sign up', new_user_path, class: 'nav-link' %>
+              <% end %>
```

Comment:
You should move this logic to the helper module.

Reply:
Thank you.